src/styles/main.scss

```diff
@@ -1,3 +1,226 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  --blue: #00acdc;
+  --white: #fff;
+  --border-color: #f3f3f3;
+  --dark: #060b35;
+  --gray: #616070;
```

Comment:
better to create varibles in :root selector
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

Reply:
![Screenshot from 2023-03-12 18-26-44](https://user-images.githubusercontent.com/33029864/224558224-35be2bd3-b4bb-4b58-a1e0-2614906656de.png)

Reply:
still not fixed